docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -15,218 +15,218 @@ menu:
+- `REDIRECT_OTHER_PORT`: **false**: If true and `PROTOCOL` is https, redirects http requests on another (https) port.
+- `PORT_TO_REDIRECT`: **80**: Port used when `REDIRECT_OTHER_PORT` is true.
 
 ## Database (`database`)
 
-- `DB_TYPE`: The database type you choose, either `mysql`, `postgres`, `mssql` or `sqlite3`.
-- `HOST`: Database host address and port.
-- `NAME`: Database name.
-- `USER`: Database username.
-- `PASSWD`: Database user password.
-- `SSL_MODE`: For PostgreSQL only.
-- `PATH`: For SQLite3 only, the database file path.
+- `DB_TYPE`: **mysql**: The database type you choose, either `mysql`, `postgres`, `mssql` or `sqlite3`.
+- `HOST`: **127.0.0.1:3306**: Database host address and port.
+- `NAME`: **gitea**: Database name.
+- `USER`: **root**: Database username.
+- `PASSWD`: **\<none\>**: Database user password.
+- `SSL_MODE`: **disable**: For PostgreSQL only.
+- `PATH`: **data/gitea.db**: For SQLite3 only, the database file path.
 
 ## Security (`security`)
 
-- `INSTALL_LOCK`: Indicates whether to allow the open install page (setting admin account is involved, so it's a very important value).
-- `SECRET_KEY`: Global secret key for your server security, **you'd better change it** (will generate a random string every time you install).
-- `LOGIN_REMEMBER_DAYS`: Cookie lifetime, in days.
-- `COOKIE_USERNAME`: Name of the cookie that saves username.
-- `COOKIE_REMEMBER_NAME`: Name of cookie that saves auto-login information.
-- `REVERSE_PROXY_AUTHENTICATION_USER`: Header name for reverse proxy authentication username.
-- `DISABLE_GIT_HOOKS`: Prevent all users (including admin) from creating custom git hooks (defaults to false)
+- `INSTALL_LOCK`: **false**: Disable to allow accessing the install page.
+- `SECRET_KEY`: **random at every install**: Global secret key. This should be changed.
+- `LOGIN_REMEMBER_DAYS`: **7**: Cookie lifetime, in days.
+- `COOKIE_USERNAME`: **gitea\_awesome**: Name of the cookie used to store the current username.
+- `COOKIE_REMEMBER_NAME`: **gitea\_incredible**: Name of cookie used to store authentication information.
+- `REVERSE_PROXY_AUTHENTICATION_USER`: **X-WEBAUTH-USER**: Header name for reverse proxy authentication.
+- `DISABLE_GIT_HOOKS`: **false**: Prevent all users (including admin) from creating custom git hooks.
 
 ## OpenID (`openid`)
 
-- `ENABLE_OPENID_SIGNIN`: Whether to allow signin in via OpenID (defaults to false).
-- `ENABLE_OPENID_SIGNUP`: Whether to allow registering via OpenID (defaults to `!DISABLE_REGISTRATION`).
-- `WHITELISTED_URIS`: Space separated list of POSIX regexp patterns. If non empty OpenID URIs should match any of these to be granted access.
-- `BLACKLISTED_URIS`: Space separated list of POSIX regexp pattenrs. OpenID URI matching any of these is refused access.
+- `ENABLE_OPENID_SIGNIN`: **false**: Allow authentication in via OpenID.
+- `ENABLE_OPENID_SIGNUP`: **! DISABLE\_REGISTRATION**: Allow registering via OpenID.
```

Comment:
here the exclamation mark has nothing to do with the importance of knowing what you're doing. Rather it's negating the `DISABLE_REGISTRATION` setting. I'm mentioning becuase I found it weird to read that meaning of the mark at the top of the file

Reply:
I think it might make the most sense to remove the use of ```:exclamation:``` since it's only used for two settings, and the mark doesn't actually render anyway. Those two options can easily have a small note added about not fiddling with them.

Reply:
you didn't get it: it's nothing about the risk in fiddling with them. Exclamation mark only means NEGATING the value of the following value. So `ENABLE_OPENID_SIGNUP` is FALSE when `DISABLE_REGISTRATION` is TRUE and vice-versa.